src/keyszer/config_api.py

```diff
@@ -41,6 +41,28 @@
 # multipurpose timeout
 _TIMEOUTS = TIMEOUT_DEFAULTS
 
+from .output import set_keystroke_delay
+
+# delay to fix Unicode entry sequence failures
+unicode_delay_ms = 0
+
+
+def throttle_delays(unicode_delay=0, keystroke_delay=0):
+    global unicode_delay_ms
+    _ud = unicode_delay
+    _kd = keystroke_delay
+    if _ud <= 100 and _ud >= 0 and isinstance(_ud, int):
+        unicode_delay_ms = _ud / 1000
+    else:
+        unicode_delay_ms = 0
+        print(f'(EE) ERROR: Unicode delay throttle must be int 1 to 100 ms. Defaulting to 0 ms.')
+    if _kd <= 150 and _kd >= 0 and isinstance(_kd, int):
+        set_keystroke_delay(_kd)
+    else:
+        set_keystroke_delay(0)
```

Comment:
I'm not sure this is necessary vs just another global... the global can use here easily be imported by `output.py`, yes?

Reply:
I have had some difficulties importing globals from other modules. But I think that was when something was changing after import, being updated by a function. I'll try it again.

Reply:
The imports should be by reference (from what I read), so it should work even if it changes after import...

Reply:
No, still having trouble actually getting the updated value of the global after config runs the API function. The delay inside `output.py` always ends up being set to the original value of zero, not the value after config injects the new values. 

Something about the order of the modules being evaluated at startup? That's why I ended up making the function that "pushes" the value into the global inside `output.py`.

Reply:
From the log output, if I print something out from `output.py`, it comes out before the line that shows the config is being evaluated. Soooo.... without the "push" function I'm not sure how to get `output.py` to recognize that the value of the global in `config_api` changed after that. 

Checking the global constantly seems excessive. These delays should only need to be set once per run.

Reply:
Moved the import of the global into `send_combo`. That makes it work. It's my understanding that imports are cached after first use, so the import shouldn't be re-done every time through, but at least it ends up with the correct value that it should be after the config file is evaluated. 

Want me to find a different way?

Reply:
It really doesn't seem to work as a direct reference link between the two modules. I also thought that was the way it would work. But it just doesn't seem to be that simple. 

I even tried initiating a new global within the scope of `output.py`, and then trying to set _NEW_GLOBAL = IMPORTED_GLOBAL, then using _NEW_GLOBAL as the argument in the `sleep_ms`. Printing out the value of either variable inside `send_combo` shows they keep the initial value of zero, the default from `config_api` before config gets the chance to inject any new values. The values never change. 

So the import of the global variable from inside `send_combo` is the only thing that has allowed `output.py` to obtain the newer value so far.

Reply:
Looks like in Python in order to get the new value of the imported variable you either need to re-import it after each time it changes in the other module (which is why importing in `send_combo` gets the newer value injected by config), or use a mutable object to store the value, such as a dict. 

Will be trying a dict to see what happens.

Reply:
OK, that seems to work. The API function will now update a globally defined dict in `config_api`. That will be imported by `output.py`. But `send_combo()` has to literally pull directly from the dict, it still can't set a global variable in `output.py` to DICT['key'] and then use the variable inside the sleep delay. So it has to be like this: 

```py
        sleep_ms(THROTTLE_DELAYS['keystroke_delay_ms'])
```

Otherwise it remains a static thing that never pulls in the new value. This somehow gets around that problem, and the import can just be at the top with all the others.

Reply:
>  it still can't set a global variable in output.py to DICT['key'] and then use the variable inside the sleep delay.

Right because that would be assign by value, not assign by reference.  Dicts work because you're importing a reference... it makes sense that simple values don't import by reference, but ChatGPT told me they would, guess it's not always right. :-)